main.go

```diff
@@ -97,6 +115,9 @@ func pointStream(csc <-chan []simpartsim.Coords) (etherdream.PointStream, error)
 		for cs := range csc {
 			dumpInPointStream(w, cs)
 		}
+		for {
```

Comment:
Also, regardless of how the repetitions are setup/controlled, I will update the existing code to be able to repeat the particle simulation.